dotCMS/src/main/java/com/dotmarketing/quartz/QuartzUtils.java

```diff
@@ -472,17 +477,17 @@ public static void scheduleTask(ScheduledTask job) throws SchedulerException, Pa
 		}
 		trigger.setMisfireInstruction(job.getMisfireInstruction());
 
-		sched.addJob(jobDetail, true);
+		scheduler.addJob(jobDetail, true);
 
 		if (isNew)
-			sched.scheduleJob(trigger);
-		else if (sched.getTrigger(triggerName, triggerGroup) != null)
-			sched.rescheduleJob(triggerName, triggerGroup, trigger);
+			scheduler.scheduleJob(trigger);
+		else if (scheduler.getTrigger(triggerName, triggerGroup) != null)
+			scheduler.rescheduleJob(triggerName, triggerGroup, trigger);

```

Comment:
Codacy found an issue: [This statement should have braces](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6213816)